src/main/java/org/skriptlang/skript/bukkit/spawner/elements/expressions/basespawner/ExprSpawnRange.java

```diff
@@ -0,0 +1,94 @@
+package org.skriptlang.skript.bukkit.spawner.elements.expressions.basespawner;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.TrialSpawner;
+import org.bukkit.event.Event;
+import org.bukkit.spawner.BaseSpawner;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spawner.SpawnerModule;
+import org.skriptlang.skript.bukkit.spawner.util.SpawnerUtils;
+
+@Name("Base Spawner - Spawn Range")
+@Description({
+	"Get the radius of the area in which the spawner can spawn entities, by default 4.",
+	"Please note that this expression gets the trial spawner configuration "
+		+ "with the current state (i.e. ominous, normal) of the trial spawner block, if one is provided.",
+	"",
+	"Base spawners are trial spawner configurations, spawner minecarts and creature spawners."
+})
+@Examples({
+	"set the spawn radius of the target block to 5",
+	"add 2 to the spawn radius of the target block",
+	"remove 1 from the spawn radius of the target block",
+	"reset the spawn radius of the target block"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.21+")
+public class ExprSpawnRange extends SimplePropertyExpression<Object, Integer> {
+
+	static {
+		register(SpawnerModule.SYNTAX_REGISTRY, ExprSpawnRange.class, Integer.class,
+			"spawn (radius|range)", "entities/blocks/trialspawnerconfigs");
```

Comment:
To prevent future conflict and/or addon conflict, should probably use `spawner spawn (radius|range)` to make this more clear.